pkg/lib/cockpit-components-firewalld-request.scss

```diff
@@ -0,0 +1,11 @@
+@import "@patternfly/patternfly/utilities/BoxShadow/box-shadow.css";
+
+.pf-c-page__main-section.ct-no-bottom-margin {
```

Comment:
This removes padding - not margin - which makes the class name misleading. No big deal however.

Reply:
Thanks for spotting! Fixed while I was re-pushing anyway.